### DISCUSSION TITLE
How to deploy Elastic Agents with GPO

### DISCUSSION THREAD

**Discussion by giomke** [Security-Onion-Solutions/securityonion#11251] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

100

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hey guys.
I am testing a new security onion in a production.
I was wondering what is the best way to distribute Elastic Agent in a domain.
Please give me your advice and experience

Thanks in advance!!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by jaypfe** [comment, marked as answer]
You can use SCCM or Intune based on what you have in your environment.  I have used them both to success. Here are a couple guides that should help!

Intune: https://practical365.com/deploying-exe-applications-with-microsoft-endpoint-manager/
SCCM: https://expertslab.wordpress.com/2014/07/06/deploy-exe-application-using-sccm-2012/#:~:text=Deploy%20EXE%20Application%20using%20SCCM%202012%201%20Copy,Choose%20%E2%80%9CManually%20specify%20the%20application%20information%E2%80%9D%2C%20Click%20Next
